kolena/_utils/datatypes.py

```diff
@@ -188,7 +222,7 @@ def deserialize_field(
             field_type = field_type or field.type
             origin = get_origin(field_type)  # non-None for typing.X types
 
-            if origin is list:
+            if origin is list or origin is Sequence:
```

Comment:
deserialization was failing for `NumberSeries = Sequence[Union[float, int]]` without it